src/components/pages/maintenance/maintenance.js

```diff
@@ -331,13 +332,20 @@ class MaintenancePage extends Component {
 
 // --- Grid events handler ends here ----------------------------------
   render() {
+    console.log(this.props, 'props needeed');
     let breadcrumbs;
     const pathName = this.props.location.pathname;
+    const alarmsGridData = this.props.alarmsGridData;
     const parentLink = <Link to={'/maintenance'}>{lang.MAINTENANCE}</Link>;
-    if (pathName.indexOf('/rule/') > 0) {
-      breadcrumbs = <span>{parentLink} &rsaquo; Alarm details</span>;
+    if (pathName.indexOf('/rule/') > 0 && (alarmsGridData || []).length > 0 ) {
+      alarmsGridData.forEach(rule => {
+      if (rule.id === this.props.params.id){
+        this.props.params.name = rule.name;
+      }
+    });
+      breadcrumbs = <span>{parentLink} <img src={ChevronRight} alt={`${ChevronRight}`} className="chevron-right" /> {this.props.params.name}</span>;
     } else if (pathName.indexOf('/job/') > 0) {
-      breadcrumbs = <span>{parentLink} &rsaquo; System status details</span>;
+      breadcrumbs = <span>{parentLink} <img src={ChevronRight} alt={`${ChevronRight}`} className="chevron-right" />{this.props.params.jobId}</span>;
```

Comment:
Don't pass ChevronRight to the alt attribute. That will just render the src path on hover. Please pass an actual description.